src/core/components/progressbar/progressbar.component.scss

```
@@ -0,0 +1,35 @@
+@import 'theme-colors';
+
+
+p{
+  text-align: center;
+  margin-bottom: 0px;
+  margin-top: 0px;
+  font-size: 1.2rem;
+  color: white;
+}
+.bar{
+  margin: 4px;
+  padding: 8px;
+}
+.desc{
+  font-size: 1rem;
+  color: grey;
+  margin-top: 0px;
+  margin-bottom: 4px;
+}
+
+.styled-p-bar{
```

```suggestion
.p-bar{
```
Ist das Prefix "styled" hier und bei der nächsten Klasse wirklich nötig?